src/modules/75-cd/components/PipelineSteps/TerraformPlan/TerraformPlan.tsx

```diff
@@ -492,7 +492,7 @@ function TerraformPlanWidget(
                             return (
                               <TFMonaco
                                 name="spec.configuration.backendConfig.spec.content"
-                                formik={formik}
+                                formik={formik as any}
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
                                formik={formik as unknown}
```